src/components/Chart/index.tsx

```diff
@@ -0,0 +1,157 @@
+import { useEffect, useMemo, useState } from "react";
+import { DetailsTooltip } from "@stellar/design-system";
+import { VictoryArea, VictoryAxis, VictoryChart, VictoryGroup } from "victory";
+import { Card } from "components/Card";
+import { formatAmount } from "helpers/formatAmount";
+import { ChartData } from "types/types.d";
+
+import "./styles.scss";
+
+type TimeFrames = { label: string; segments: number | undefined }[];
+
+interface ChartProps {
+  chartData: ChartData[];
+  colorRgb: number[];
```

Comment:
to follow the design, we need to use an rgb color to generate two different rgba colors. Wasn't quite sure what the best type is here, so trying an array of the rgb values.